### ISSUE TITLE
Wrong version number on npm / releases

### ISSUE THREAD

**Issue by tomByrer** [cloudflare/wrangler#306]
Win10
```bash
$ npm install -g @cloudflare/wrangler
C:\Program Files\nodejs\wrangler -> C:\Program Files\nodejs\node_modules\@cloudflare\wrangler\run-wrangler.js

> @cloudflare/wrangler@1.0.2 postinstall C:\Program Files\nodejs\node_modules\@cloudflare\wrangler
> node install-wrangler.js

Downloading release https://github.com/cloudflare/wrangler/releases/download/v1.0.0/wrangler-v1.0.0-x86_64-pc-windows-msvc.tar.gz
Wrangler has been installed!
+ @cloudflare/wrangler@1.0.2
added 27 packages from 13 contributors in 5.711s

tomby@DESKTOP-9KQB7OF MINGW64 ~/OneDrive/Documents/github/learn/cfwk
$ wrangler --help
 wrangler 1.0.0
(etc)
```

https://www.npmjs.com/package/@cloudflare/wrangler = `v1.0.2`

Thanks for all your work!

**Comment by EverlastingBugstopper** [comment]
I think this is an issue with some release process. AFAIK we haven’t officially updated wrangler past 1.0.0, we have a PR open for 1.1.0. Perhaps a one-off `npm run publish` was ran at some point. Perhaps @ashleygwilliams can shed some light

**Comment by EverlastingBugstopper** [comment]
@tomByrer I did some investigating! It turns out that our npm package and our wrangler binary have different release schedules. The package we have on npm is just an installer that fetches the latest release of wrangler and installs it on your machine. They’re two separate pieces of software! (This was unclear to me at first). 

So to recap: the `v1.0.2` you see is the version of wrangler’s npm installer, and the `wrangler 1.0.0` is the version of wrangler you have installed. 

Does this make sense? Do you think we should clarify this somewhere/maybe make a note in our installation docs?

**Comment by tomByrer** [comment]
That is confusing.

Install doc notes would help.  Perhaps calling the npm package "wrangler installer" would be better, since that what it is, & not the "wrangler CLI"?

**Comment by EverlastingBugstopper** [comment]
Thanks for your response @tomByrer -- 

I don't think that renaming the npm package is the way to go here, _however_ I agree that this is confusing. We're going to add an explanation for this on the install docs page -- I'll mention you on the PR when we get it up!

Thanks again :)

**Comment by ashleymichal** [comment]
I believe this issue will be solved by the fix outlined in this rfc: #380

**Comment by ashleygwilliams** [comment]
this is now fixed in 1.1.1! #381
